src/amber/router/params.cr

```diff
@@ -34,7 +35,7 @@ module Amber::Router
 
     def files
       multipart unless @multipart
-      @files
+      @files.not_nil!
```

Comment:
Do we want an exception if this is nil or should it return an empty array?

Reply:
That's a good idea.